pkg/controller/certificates/issuing/internal/keystore_test.go

```diff
@@ -398,13 +398,23 @@ func TestManyPasswordLengths(t *testing.T) {
 	certPEM := mustSelfSignCertificate(t, nil)
 	caPEM := mustSelfSignCertificate(t, nil)
 
+	const testN = 10000
+
 	// We will test random password lengths between 0 and 128 character lengths
 	f := fuzz.New().NilChance(0).NumElements(0, 128)
+	// Pre-create password test cases. This cannot be done during the test itself
+	// since the fuzzer cannot be used concurrently.
```

Comment:
suggestion (non-blocking): is it worth adding a note here that if gofuzz gets a release newer than v1.2.0 this should be usable in parallel? Likely nobody is gonna make that change after we merge this though, so maybe it's not worth it